pulser/sequence.py

```diff
@@ -39,16 +39,16 @@
 from pulser.register import Register
 from pulser._seq_drawer import draw_sequence
 
-if version_info[:2] == (3, 7):  # pragma: no cover
+if version_info[:2] >= (3, 8):  # pragma: no cover
+    from typing import Literal, get_args
+else:  # pragma: no cover
     try:
-        from typing_extensions import Literal, get_args
+        from typing_extensions import Literal, get_args  # type: ignore
```

Comment:
If you're changing this here, can you change it in `simconfig.py` too please?

Reply:
I wasn't aware of that one. Done.